abeja/logging.py

```diff
@@ -0,0 +1,10 @@
+import logging
```

Comment:
If you want to keep it separate from `abeja.common.logging.logger` because it is a custom logger, `abeja.tracking.logging.logger` is better because `abeja.logging.logger` is more global.

Reply:
Instead of defining `abeja.tracking.logging.logger`, define logger in `abeja.tracking.__init__.py` because it is tracking-specific logger for now.